src/scss/includes/reviewScore.scss

```diff
@@ -1,10 +1,11 @@
 .reviewScore {
```

Comment:
The `<ReviewScore>` component is also used in popups for PJ POIs. Have you checked this doesn't change the look too much?

Reply:
Yes it seems ok to me

Reply:
![image](https://user-images.githubusercontent.com/2981774/83411852-4f4ff800-a419-11ea-9f2d-2e4197a39086.png)

Reply:
I meant the map popups on, not the cards.
![Capture d’écran de 2020-06-01 18-14-15](https://user-images.githubusercontent.com/243653/83429097-d27e4780-a433-11ea-80b6-07019f2ea4a1.png)

But it looks ok anyway :+1: